### PR TITLE
Adjust header component

### PR DESCRIPTION
**Pull request by sergeyschegolev**

### CODE REVIEW

File: src/Header/Header.tsx [hunk 1]
```diff
@@ -102,19 +109,21 @@ export class Header extends React.Component<HeaderProps, HeaderState> {
...
-              <DropdownContainer>
-                <UserMenu
-                  userName={userName}
-                  email={email}
-                  onLogout={onLogout}
-                />
-              </DropdownContainer>
-            </Dropdown>
+          {isAuth ? (
+            userName ? (
```
Comment (PaulRumyantsevItr) [inline src/Header/Header.tsx]: Move to separate render method

File: src/Header/style.scss [hunk 2]
```diff
@@ -45,7 +46,7 @@
         background-image: url('../img/pay-icn.svg');
       }
 
-      &_profile {
+      &_settings {
```
Comment (PaulRumyantsevItr) [inline src/Header/style.scss]: Shouldn't it be __settings?

File: src/__tests__/Header.spec.tsx [hunk 3]
```diff
@@ -13,7 +13,32 @@ export const headerLinkOptions = [
...
+    title: MenuItem.Settings,
+    url: '#'
+  }
+];
+
+export const secondaryMenuItems = [
+  {
+    iconName: 'lykke-streams',
+    title: MenuItem.LykkeStreams,
+    url: '#'
```
Comment (PaulRumyantsevItr) [inline src/__tests__/Header.spec.tsx]: I see that iconName is not required, why can't we do the same with url?

File: src/Header/style.scss (hunk 2 above)
Comment (sergeyschegolev) [reply src/Header/style.scss]: No

File: src/__tests__/Header.spec.tsx (hunk 3 above)
Comment (sergeyschegolev) [reply src/__tests__/Header.spec.tsx]: Because it is required in all cases and '#' is just a mock

**Pull request merged**